packages/builder/src/components/design/settings/controls/ColumnEditor/ColumnEditor.svelte

```diff
@@ -32,6 +32,12 @@
     allowLinks: true,
   })
 
+  $: {
+    value = (value || []).filter(column =>
+      Object.keys(schema || {}).includes(column.name || column)
```

Comment:
Could you not do `(schema || {})[column.name || column] !== undefined` here? Thinking this because `.keys` and `.includes` are both O(n) operations.

Reply:
Nice! Done.